libpromises/evalfunction.c

```diff
@@ -8879,6 +8884,9 @@ void ModuleProtocol(EvalContext *ctx, const char *command, const char *line, int
 
             StringSetAddSplit(tags, content, ',');
             StringSetAdd(tags, xstrdup("source=module"));
+
+            xasprintf(&derived_from, "derived_from=%s", command);
+            StringSetAdd(tags, derived_from);
```

Comment:
Would be nice to have `StringSetAddF(tags, "derived_from=%s", command);`, but it's a PITA to add in the *libntech* submodule.

Reply:
Indeed https://github.com/cfengine/libntech/pull/182